installer/manifests/keptn/templates/_helpers.tpl

```diff
@@ -465,6 +465,7 @@ Usage:
 
 {{- define "keptn.initContainers.wait-for-nats" -}}
 - name: "wait-for-nats"
+  namespace: "{{ .Release.Namespace }}"
```

Comment:
this is not needed here since the init container is filled directly into the pod.spec.initConainers field